prime-router/src/main/kotlin/azure/DatabaseAccess.kt

```diff
@@ -166,11 +164,20 @@ class DatabaseAccess(private val create: DSLContext) : Logging {
      * ActionHistory queries
      */
 
-    fun fetchReportFile(reportId: ReportId, txn: DataAccessTransaction? = null): ReportFile {
+    /**
+     * You should include org as a search criteria to enforce authorization to get that report.
+     */
+    fun fetchReportFile(reportId: ReportId, org: Organization? = null, txn: DataAccessTransaction? = null): ReportFile {
```

Comment:
This addition is to support DownloadFunction using REPORT_FILE rather than TASK to get the file to download.